### PR TITLE
about and contact pages created

### PR DESCRIPTION
**Pull request by SauleJ**
About and contact pages created

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,320 @@
+{
```
Comment (Ech0ke) [inline package-lock.json:1]: remove this file from commit and from your local storage. It is not needed.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (Ech0ke) [inline package.json:1]: remove this file from commit and from your local storage. It is not needed.

File: .vscode/launch.json [hunk 3]
```diff
@@ -0,0 +1,35 @@
+{
```
Comment (Ech0ke) [inline .vscode/launch.json]: remove this file and folder that the file lives in from commit and from your local storage. It is not needed.

File: .vscode/tasks.json [hunk 4]
```diff
@@ -0,0 +1,41 @@
+{
```
Comment (Ech0ke) [inline .vscode/tasks.json]: remove this file and folder that the file lives in from commit and from your local storage. It is not needed.

File: .vscode/launch.json (hunk 3 above)
Comment (Matas-V) [inline .vscode/launch.json]: I guess these files got generated as you build the whole project (FE and BE) from VS code IDE, however these files should not be deployed to the whole project, so you could add this line `.vscode` in the file `.gitignore` and those files should be ignored and not pushed to the branch.

File: fe/src/app/about/page.tsx [hunk 5]
```diff
@@ -0,0 +1,44 @@
+import React from 'react';
```
Comment (Matas-V) [inline fe/src/app/about/page.tsx]: Could be removed.

File: fe/src/app/globals.css [hunk 6]
```diff
@@ -1,3 +1,8 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
```
Comment (Matas-V) [inline fe/src/app/globals.css]: Please undo these new lines.

File: fe/src/app/about/page.tsx [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+import Image from 'next/image';
+
+const AboutUsPage = () => {
+  return (
+    <div className="bg-gray-800 text-white">
+      <main className="py-15 text-center">
+        <div className="relative overflow-hidden w-full h-96 rounded-lg shadow-lg mb-8">
+            <Image
```
Comment (Matas-V) [inline fe/src/app/about/page.tsx]: Heard from Aidas that there is a bug in next realted to images, so I suggest using simple `` tag and just adding the src as the link of image from the internet.

File: fe/src/app/about/page.tsx [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+              src="/public/images/about-us.jpg"
+              alt="About Us Image"
+              layout="fill"
+              className="rounded-lg transition-transform transform hover:scale-105 duration-300"
```
Comment (Matas-V) [inline fe/src/app/about/page.tsx]: The `rounded-lg` is not needed in my opinion.

File: fe/src/app/contact/page.tsx [hunk 9]
```diff
@@ -0,0 +1,73 @@
...
+        <h1 className="text-7xl md:text-7xl font-bold mb-12">Contact Us</h1>
+        <div className="max-w-2xl mx-auto text-xl md:text-2xl leading-relaxed">
+          <p className="mb-12">
+            We{'\''}re here to assist you! Feel free to reach out if you have any questions, suggestions, or just want to say hello.
+          </p>
+          <p className="mb-12">
+            Connect with us through the following channels:
+          </p>
+          <ul className="list-disc list-inside mb-8">
+            <li>Email: <a href="mailto:info@example.com" className="text-blue-500 hover:underline">CarFans@example.com</a></li>
```
Comment (Matas-V) [inline fe/src/app/contact/page.tsx]: For me, those bullet points look ugly, we can remove them with this property `list-none` for ul tag. Also I would not mind if we aligned them in left not center like now.

image

Comment (Matas-V) [reply fe/src/app/contact/page.tsx]: Also not sure if we need link for email.

File: fe/src/app/contact/page.tsx [hunk 10]
```diff
@@ -0,0 +1,73 @@
...
+            crossOrigin=""
+          />
+          <Button
+            type="button"
+            color="blue"
+            size="lg"
+            className="rounded"
+          >
+            Send Message
+          </Button>
```
Comment (Matas-V) [inline fe/src/app/contact/page.tsx:133]: I suggest making at least fake request sending like onClick we would clear all fields and show the toast message of succesfully send letter.

Comment (Matas-V) [reply fe/src/app/contact/page.tsx:133]: Also, button styling should follow our styling across whole app, so add variant="gradient" with color black, which is by default.

**Review by Matas-V — CHANGES_REQUESTED** [review]

**Review by Ech0ke — APPROVED** [review]
Fixed input styles, resolved conflicts

**Pull request merged**